src/wdio/services/AxeService.js

```diff
@@ -0,0 +1,47 @@
+/* global browser, axe */
+import fs from 'fs';
+
+let axeCoreSrc = fs.readFileSync('node_modules/axe-core/axe.min.js', 'utf8');
```

Comment:
I would make this fs.readFileSync(path.join(__dirname, 'node_modules', 'axe-core', 'axe.min.js', 'utf8'))

Reply:
I edited that a few times and there may be a better alternative.  We may want to play around with making sure this is available in various development scenarios (hitting terra-toolkit locally, from git, etc.)